### PR TITLE
feat: add talk title to Speakers & Grid Schedule in Topics

### PR DESCRIPTION
**Pull request by vannyle**
**Describe what changes this pull request brings**
- For Hero:
- [x] change title 'Summit 2021' to 'eBPF Summit 2021'
- For Featured Speakers:
- [x] add talk titles
- [x] add Sessionize page for each speaker
- For Topics:
- [x] replace the Topics section with an embedded Schedule Grid from Sessionize
- For Agenda:
- [x] replace link with a button

**Describe any additional changes**

**Steps to test**
1. Visit [the page](https://deploy-preview-2--ebpf-summit-2021.netlify.app/summit-2021/)
2. Confirm that everything looks and works as expected

**Screenshots**
Featured Speakers:
image
Agenda:
image
Schedule:
image

**Checklist**

- [x] I've tested everything and everything works well
- [x] I've tested these changes in other browsers
- [x] I've checked these changes on all breakpoints

### CODE REVIEW

File: src/components/pages/home/topics/schedule/schedule.jsx [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+const SESSIONIZE_URL = 'https://sessionize.com/api/v2/f3vhdjzc/view/GridSmart';
+
+const Schedule = ({ className }) => {
+  const [scheduleHTML, setScheduleHTML] = useState(null);
+  useEffect(() => {
+    fetch(SESSIONIZE_URL)
+      .then((res) => res.text())
+      .then((htmlText) => setScheduleHTML(htmlText));
+  }, []);
+  return <div className={className} dangerouslySetInnerHTML={{ __html: scheduleHTML }} />;
```
Comment (BolotskyDev) [inline src/components/pages/home/topics/schedule/schedule.jsx:13]: Let's return `null` untill `scheduleHTML` is there

File: gatsby-ssr.js [hunk 2]
```diff
@@ -1,7 +1,14 @@
...
+  <script type="text/javascript" src="https://sessionize.com/api/v2/yp5syx3f/view/GridSmart" />,
+];
+
+const SUMMIT_PAGE_NAME = '/summit-2021/';
+
+exports.onRenderBody = ({ pathname, setPostBodyComponents }) => {
+  if (pathname === SUMMIT_PAGE_NAME) {
+    setPostBodyComponents(postBodyComponents);
+  }
+};
```
Comment (BolotskyDev) [inline gatsby-ssr.js:14]: revert the changes in this file as they are no longer needed

File: src/components/pages/home/topics/topics.jsx [hunk 3]
```diff
@@ -1,35 +1,15 @@
...
+      <Schedule />
     </Container>
   </section>
 );
 
-Topics.propTypes = {
-  title: PropTypes.string.isRequired,
-  items: PropTypes.arrayOf(PropTypes.string).isRequired,
-  text: PropTypes.string.isRequired,
-};
```
Comment (BolotskyDev) [inline src/components/pages/home/topics/topics.jsx:33]: Since it is now more of a `Schedule` then `Topics`, rename all the necessary files and avoid creating unnecessary abstractions. 

Basically, let's make `topics.jsx` -> `schedule.jsx` and put all the logic in `schedule.jsx`

**Review by BolotskyDev — CHANGES_REQUESTED** [review]

File: gatsby-ssr.js (hunk 2 above)
Comment (vannyle) [reply gatsby-ssr.js]: Thank you for pointing that out!
